deploy/docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM alpine:3.5
+
+LABEL maintainer="adundovi@gmail.com"
+LABEL description="Prepares a docker image \
```

Comment:
Pretty sure this should be a description of the built image itself, so please replace with `Stream server that does most of what people really want from a social network.`

We should probably change that to something less jargon-y, to be honest. But in the meantime...